jscomp/common/js_config.mli

```diff
@@ -89,6 +89,9 @@ val sort_imports : bool ref
 val syntax_only  : bool ref
 val binary_ast : bool ref
 
+#if BS_NATIVE then
+val simple_binary_ast : bool ref
+#end
 
 val bs_suffix : bool ref
 val debug : bool ref
```

Comment:
This does not have to be on conditional compilation, we can support it natively in a separate PR